src/d3d_chords.pro

```diff
@@ -53,14 +53,16 @@ FUNCTION oblique_geo_from_patch, patchfile, skip=skip
         f = patchdat.(i).fibers
         for j=0,n_elements(f.fibers)-1 do begin
             fnum = f.fibers[j]
+            name = 'p'+string(ceil(float(fnum)/3),FOR='(I02)')+suffix[fnum mod 3 -1]
+            IF N_ELEMENTS(id) && TOTAL(STRCMP(name,id)) THEN CONTINUE
```

Comment:
What is the purpose of this change, again?

Reply:
Each chord contains information for the whole row of fiber. If two different chords reside on the same row, redundant fiber geometry will be appended. This will check if the fiber information has been already added.